src/services/CalculatorProcess.js

```diff
@@ -1,51 +1,41 @@
-let memory = [];
-let result = "";
-function calculatorProcess(content,visor) {
-	
-	const numbersOperators = () => String(eval((result)));
-	
-	memory += content;
+let memory = "", process = "";
+let sLenth = 0;
 
-	if (memory.includes("=")) {
-
-		result = memory.slice(0,-1);
-
-		if (memory.includes("+")) {
-			memory = numbersOperators();
-		}
+function CalculatorProcess(content, visor) {
+	const PLUS_MINUS_SIGNAL = "S";
+	const convertion = (string) => String(eval(string))
+	const firstInterations = ["S", "+", "-", "/", "%", "*", "="];
+	const clear = () => process = memory = "";
 
-		if (memory.includes("-")) {
-			memory = numbersOperators();
-		}
+	memory += content
 
-		if (memory.includes("*")) {
-			memory = numbersOperators();
-		}
+	if (firstInterations.includes(memory) && memory.length < 2 || content === "C") {
```

Comment:
It's possible to avoid this

```
memory.length < 2
```

Reply:
i didn't understand that it'd work without this, i got it the reason after i tested

Reply:
done